Shared/Samples/Display device location with NMEA data sources/README.metadata.json

```
@@ -0,0 +1,39 @@
+{
+    "category": "",
+    "description": "Parse NMEA sentences and use the results to show device location on the map.",
+    "ignore": false,
+    "images": [
+        "display-device-location-with-NMEA-data-sources.png"
+    ],
+    "keywords": [
+        "Bluetooth",
+        "GNSS",
+        "GPS",
+        "NMEA",
+        "accessory",
+        "history",
+        "navigation",
+        "real-time",
+        "trace",
+        "Location",
+        "LocationDisplay",
+        "NMEALocationDataSource",
+        "NMEASatelliteInfo"
+    ],
+    "offline_data": [
+        "d5bad9f4fee9483791e405880fb466da"
+    ],
+    "redirect_from": [],
+    "relevant_apis": [
+        "Location",
+        "LocationDisplay",
+        "NMEALocationDataSource",
+        "NMEASatelliteInfo"
+    ],
+    "snippets": [
+        "DisplayDeviceLocationWithNMEADataSourcesView.swift",
+        "DisplayDeviceLocationWithNMEADataSourcesView.Model.swift",
+        "SimulatedNMEADataSource.swift"
```

```suggestion
        "SimulatedNMEAData.swift"
```
To get rid of the status check error.